### PR TITLE
ROS node for hand gesture recognition

### PR DESCRIPTION
**Pull request by katerynaCh**
This PR implements a ROS node for rgbd_hand_gesture_learner.

### CODE REVIEW

File: projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+        # concatenate and standardize
+        img = np.concatenate([image, np.expand_dims(depth_img, axis=-1)], axis = -1)
+        img = (img - self.mean) / self.std
+        
+        img = Image(img, dtype=np.float32)
+        return img
+
+
+def from_category(prediction, source_data=None):
```
Comment (passalis) [inline projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py:123]: Please move this generic functionality to [projects/opendr_ws/src/ros_bridge/src/opendr_bridge/bridge.py](https://github.com/tasostefas/opendr_internal/blob/master/projects/opendr_ws/src/ros_bridge/src/opendr_bridge/bridge.py).

**Review by passalis — CHANGES_REQUESTED** [review]
Thank you! Overall seems fine. Please move the generic translation functionality into the bridge package, as well as please add the name of the node into the list of available nodes in [](https://github.com/tasostefas/opendr_internal/blob/master/projects/opendr_ws/README.md)

File: projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py (hunk 1 above)
Comment (passalis) [reply projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py:123]: Also, after moving the corresponding function to bridge.py please update the corresponding documentation (https://github.com/tasostefas/opendr_internal/blob/master/docs/reference/rosbridge.md).

**Review by passalis — DISMISSED** [review]
Thank you!

**Review by ad-daniel — DISMISSED** [review]
Thank you!

**Comment by ad-daniel** [comment]
Looks like there are a few formatting fixes that need to be done before being able to merge

**Review by passalis — APPROVED** [review]

**Pull request merged**